superset-frontend/src/components/Chart/DrillBy/useDisplayModeToggle.tsx

```diff
@@ -19,49 +19,38 @@
 
 import { useMemo, useState } from 'react';
 import { css, SupersetTheme, t } from '@superset-ui/core';
-import { Radio } from 'src/components/Radio';
+import { Radio, CheckboxOptionType } from 'src/components/Radio';
 import { DrillByType } from '../types';
 
 export const useDisplayModeToggle = () => {
   const [drillByDisplayMode, setDrillByDisplayMode] = useState<DrillByType>(
     DrillByType.Chart,
   );
-
+  const customButtons: CheckboxOptionType[] = [
```

Comment:
please either wrap it in useMemo or move it out of the hook (it has only static values so should work fine) - otherwise it will trigger rerenders on every render cycle